shims/common/src/main/scala/io/glutenproject/GlutenConfig.scala

```diff
@@ -492,10 +492,10 @@ object GlutenConfig {
       .booleanConf
       .createWithDefault(true)
 
-  val VANILLA_COLUMNAR_READERS_ENABLED =
-    buildConf("spark.gluten.sql.columnar.vanillaReaders")
+  val VANILLA_VECTORIZED_READERS_ENABLED =
+    buildConf("spark.gluten.sql.columnar.enableVanillaVectorizedReaders")
```

Comment:
Removing "s" looks better:  `enableVanillaVectorizedReader`. How do you think?

Reply:
It may include several fileformat's reader, such as orc, parquet, hope make sense for you.

Reply:
Makes sense.